$NetBSD: patch-Swiften_SConscript,v 1.1 2022/05/03 20:03:31 nia Exp $

Reproducible Builds: Sort directory contents at build. via
Arch Linux.

--- Swiften/SConscript.orig	2018-04-06 10:06:46.000000000 +0000
+++ Swiften/SConscript
@@ -567,7 +567,9 @@ if env["SCONS_STAGE"] == "build" :
     swiften_includes = []
     swiften_public_includes = []
     top_path = env.Dir("..").abspath
-    for root, dirs, files in os.walk(env.Dir(".").abspath) :
+    for root, dirs, files in sorted(os.walk(env.Dir(".").abspath)) :
+        dirs.sort()
+        files.sort()
         if root.endswith("UnitTest") :
             continue
         for file in files :
